example/src/main/java/com/stripe/example/activity/PaymentAuthActivity.java

```diff
@@ -46,6 +49,8 @@
      */
     private static final String PAYMENT_METHOD_3DS2_REQUIRED = "pm_card_threeDSecure2Required";
     private static final String PAYMENT_METHOD_3DS_REQUIRED = "pm_card_threeDSecureRequired";
+    private static final String PAYMENT_METHOD_AUTH_REQUIRED_ON_SETUP =
```

Comment:
unrelated to this patch, but I notice `PAYMENT_METHOD_3DS_REQUIRED` isn't used anywhere; should it be deleted?

Reply:
I believe it was left in for easily switching between 3DS and 3DS2 when testing those payment auth flows.